js/editor-cm.js

```diff
@@ -8,20 +111,30 @@ var EditSession = CodeMirror.Doc;
 function EditorCodeMirror(editorElement, settings) {
   this.element_ = editorElement;
   this.settings_ = settings;
+
+
```

Comment:
nit: extra blank line. no need for either blank line here imo

Reply:
Done